src/graphnet/models/task/reconstruction.py

```diff
@@ -141,14 +141,6 @@ def _forward(self, x: Tensor) -> Tensor:
 class TimeReconstruction(Task):
     """Reconstructs time."""
 
-    # Requires one feature, time.
```

Comment:
I suggest you revert this change — it is not pertinent to the goal of the PR.

Reply:
I am really confused about why this happened, as I do not recall changing this function. I was able to locate the change to commit ["33beeb97d86e3afd46bd8b245bd9592504159911" ](https://github.com/MortenHolmRep/graphnet/commit/33beeb97d86e3afd46bd8b245bd9592504159911) during a merge from main.

Reply:
The function has been restored.